GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -20,7 +20,9 @@ public ConfigFileSettings(ConfigFileSettings lowerPriority, ConfigFileSettingsCa
 
         public static ConfigFileSettings CreateEffective(GitModule module)
         {
-            return CreateLocal(module, CreateGlobal(CreateSystemWide()));
+            var result = CreateLocal(module, CreateGlobal(CreateSystemWide()));
+            result.SettingLevel = SettingLevel.Effective;
+            return result;
```

Comment:
The submission exhibits two different ways of initialising settings.
One way is via an intermediately variable (like here) and another is without.
Please settle on a single style and use it consistently.